.github/actions/notify-lula/action.yaml

```diff
@@ -0,0 +1,51 @@
+name: Notify Lula
+description: "Comment on PR to notify Lula Team"
+
+
+inputs:
+  state:
+    description: 'state of the comment update'
+    required: true
+    default: ''
+  flavor:
+    description: 'flavor of the comment update'
+    required: true
+    default: ''
+  ghToken:
+    description: 'GITHUB_TOKEN'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Find Comment
+      uses: peter-evans/find-comment@3eae4d37986fb5a8592848f6a574fdf654e61f9e # v3.1.0
+      id: fc
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        comment-author: 'github-actions[bot]'
+        body-includes: Compliance ${{ inputs.flavor }} Evaluation
+        token: ${{ inputs.ghToken }}
+
+    - name: Create comment
+      if: ${{ steps.fc.outputs.comment-id == '' && inputs.state == 'failure'}}
+      uses: peter-evans/create-or-update-comment@71345be0265236311c031f5c7866368bd1eff043 # v4.0.0
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        token: ${{ inputs.ghToken }}
+        body: |
+          Compliance ${{ inputs.flavor }} Evaluation: ${{ inputs.state }}
+
+          CC: @defenseunicorns/lula-dev
+
+    - name: Update comment
+      if: ${{ steps.fc.outputs.comment-id != '' }}
+      uses: peter-evans/create-or-update-comment@71345be0265236311c031f5c7866368bd1eff043 # v4.0.0
+      with:
+        comment-id: ${{ steps.fc.outputs.comment-id }}
+        token: ${{ inputs.ghToken }}
+        edit-mode: replace
+        body: |
+          Compliance ${{ inputs.flavor }} Evaluation: ${{ inputs.state }}
+
+          CC: @defenseunicorns/lula-dev
```

Comment:
What do you think about making it more clear that this is an update to a previous comment? At first glance [this comment](https://github.com/defenseunicorns/uds-core/pull/496#issuecomment-2226516061) feels weird that the unicorn flavor is called out but the others are not - wondering if a reaction or appending to the comment makes more sense?

Definitely a nit and would defer to whatever helps your team most.

Reply:
Yeah I considered more complex workflows for appending. You see the unicorn comment by itself because the other two live persistently up further in the commits. 

I'm open to suggestions here - it could be one comment that gets edited / appended / etc - it could be multiple as it is right now.

Mainly I wanted to balance complexity and confusion - so there still may be work to do here.

Current process:
- Comment only appears after an evaluation has failed
- One comment will be made (per flavor) for the life of a PR
- That comment will be updated on each subsequent run (whether pass or fail)
- This signifies the current state

Reply:
Let's leave it as-is for now and iterate if needed unless you can think of a simple change here.

I think the main thing that feels a bit weird is that comments only appear on failure, but if the checks later pass you end up with a comment that says "Success" without the history of why that comment is there (i.e. because it previously failed).